services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -1841,6 +1841,25 @@ module.exports = {
       return c;
     },
 
+    /**
+     *
+     */
+    contentCompanyField: async (_, { input }, { base4rest, basedb }, info) => {
+      validateRest(base4rest);
+      const company = 'platform/content/company';
+      const { id, companyId } = input;
+      const content = await basedb.strictFindById('platform.Content', id, { projection: { type: 1 } });
+      const model = `platform/content/${dasherize(content.type)}`;
+      const body = new Base4RestPayload({ type: model });
+      if (companyId) body.setLink('company', { id: companyId, type: company });
+      if (content) {
+        body.set('id', id);
+        await base4rest.updateOne({ model, id, body });
+      }
```

Comment:
`id` is not nullable, and `strictFindById` will error before reaching this line if a bad ID is passed. No need for the if check.